mod/caching/set/type_plus_right/metric/source.rb

```diff
@@ -15,7 +15,7 @@ def cache_query?
 end
 
 def answer_ids
-  Answer.where(metric_id: left.id).where.not(answer_id: :nil).pluck :answer_id
+  ::Answer.where(metric_id: left.id).where.not(answer_id: :nil).pluck :answer_id
```

Comment:
Is colon nil really the right syntax?

Reply:
ooh, I don't think so.  I suspect this is a(n old) bug.